integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -155,6 +155,43 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 				ContainElement(ContainSubstring("vpc-cni")),
 			))
 
+			By("Creating a fargate profile")
+			cmd = params.EksctlCreateCmd.
+				WithArgs(
+					"fargateprofile",
+					"--cluster", clusterName,
+					"--name", "fp-test",
+					"--namespace", "default",
+					"--verbose", "2",
+				)
+			Expect(cmd).To(RunSuccessfullyWithOutputStringLines(
+				ContainElement(SatisfyAll(ContainSubstring("created"), ContainSubstring("fp-test"))),
+			))
```

Comment:
I could be mis-emembering, but doesn't something have to change in the way you setup a cluster for fargate to work?

Reply:
That's why I needed to add the private subnets.

Reply:
ah my bad I was looking directly at the integration commit, missed that separate commit 😄